sdk/identity/azure-identity/src/main/java/com/azure/identity/BrowserCustomizationOptions.java

```diff
@@ -7,31 +7,31 @@
  * Represent Options to customize browser view.
  */
 public class BrowserCustomizationOptions {
-    private String htmlMessageSuccess;
-    private String htmlMessageError;
+    private String successMessage;
+    private String errorMessage;
 
     /**
-     * Configures the property to set HtmlMessageSuccess which the browser will show to the user when the user
-     * finishes authenticating successfully.
+     * Configures the property to set the success message which the browser will show to the user upon successful
+     * authentication.
      *
-     * @param htmlMessageSuccess the message to display when user finishes authenticating.
+     * @param successMessage the message to display when user finishes authenticating.
      * @return the updated options.
      */
-    public BrowserCustomizationOptions setHtmlMessageSuccess(String htmlMessageSuccess) {
-        this.htmlMessageSuccess = htmlMessageSuccess;
+    public BrowserCustomizationOptions setSuccessMessage(String successMessage) {
+        this.successMessage = successMessage;
         return this;
     }
 
     /**
-     * Configure the property to set HtmlMessageError which the browser will show to the user when the user
-     * finishes authenticating, but an error occurred. You can use a string format e.g.
+     * Configure the property to set error message which the browser will show to the user upon failed authentication.
```

Comment:
can be changed to upon failure to acquire an access token.
If user doesn't authenticate successfully in browser, msal will never get auth code to fetch an access token.
This error message will pop up once user authenticates in browser successfully, but we fail to get an access token after that.